fabric-common/lib/impl/bccsp_pkcs11.js

```diff
@@ -599,8 +599,7 @@ class CryptoSuite_PKCS11 extends CryptoSuite {
 			 * Error will be thrown if signature is not properly encoded.
 			 */
 			const rns = new ecsig(signature, 'hex');
-			logger.debug(__func() + 'ECDSA R+S signature: ' +
-				util.inspect(rns, {depth: null}));
+			logger.debug(__func() + 'ECDSA R+S signature: ', rns);
```

Comment:
This code change aim to solve deprecation warning
`(node:6461) [DEP0079] DeprecationWarning: Custom inspection function on Objects via .inspect() is deprecated`